spi/src/main/java/org/n52/io/request/IoParameters.java

```diff
@@ -586,7 +586,13 @@ private BBox createBbox() {
             return null;
         }
         String bboxValue = getAsString(BBOX);
-        BBox bbox = parseJson(bboxValue, BBox.class);
+        BBox bbox;
+        // Check if supplied in minx,miny,maxx,maxy format - else assume json
+        if (bboxValue.matches("^(\\d*\\.?\\d*\\,){3}(\\d*\\.?\\d*)$")) {
+            bbox = new BBox(bboxValue);
```

Comment:
Could you move parsing `GeojsonPoint`s to here, please? Otherwise the `public BBox(String)` constructor is too loose when accepting just a `String` ... if only lowerLeft and upperRight are parsed you could reuse the already existing constructor then.